eras/shelley/test-suite/bench/Main.hs

```diff
@@ -375,10 +386,19 @@ varyDelegState tag fixed changes initstate action =
 
 -- =============================================================================
 
-main :: IO ()
+main :: forall era. 
+  ( EraGen era,
+    Core.PParams era ~ ShelleyPParams era,
+    Mock (EraCrypto era),
+    Embed (Core.EraRule "DELPL" era) (CERTS era),
+    Environment (Core.EraRule "DELPL" era) ~ DelplEnv era,
+    State (Core.EraRule "DELPL" era) ~ DPState (EraCrypto era),
+    Signal (Core.EraRule "DELPL" era) ~ DCert (EraCrypto era),
+    ShelleyTest era
+  ) => IO ()
 -- main=profileValid
```

Comment:
What is all this about?

The only sensible signature for `main` is `main :: IO ()` because you can never provide any argument or type annotation when running a process

Reply:
Thanks Alexey! That makes sense. I will remove this.